ChangeLog.md

```diff
@@ -3,6 +3,308 @@
+characters in game
+- Added a right-click command to dismiss sensor ghosts on the fleets
+window, which causes an empire to forget a fleet existed, removing it
+from the map.
+- Disabled sound system init when sound and music are disabled
+- Fix crash when OpenAL device does not initialize
+- Added message box at startup informing player in case of audio
+system init failiure
+- Added message box at startup informing player in case of
+insufficient OpenGL version
+- Made pressing escape close more GUI windows
+- Added tech turn and RP requirements to tech panels on tech tree
+- Modified autogenerated fleet names to better reflect fleet contents
+- Increase fleet drop target width and add an icon to it
+- Modified how client and server communicate in a single player game
+to avoid firewall popups about network access
+- Added scanlines over fleets when not currently visible
+- Added nearly-accurate estimate of meter changes on the next turn to
+meter tooltips
+- Made MultiMeterStatusBar rescale to show largest value
+- In DesignWnd add ability to drag parts away from design
+
+
+#### Content
+
+- Translation updates: French, Russian
+- Removed / updated various missing, unused, or incorrect stringtable entries
+- Adjusted AI priorities for various buildings
+- Adjusted AI colonization planning
+- Made AI more cautious of hidden threats after losing fleets in a
+system it can't detect into
+- Made AI adjust priorities based on costs of ship parts and hulls
+- Adjusted AI priorities for research time
+- Adjusted AI industry modifiers
+- Adjusted AI weapon upgrade calculations
+- Updated pedia descriptions of various content and UI windows,
+including adding more links
+- Reworked intro sitrep to be less immersion-breaking
+- Moved start-of-game unlocking sitreps to before-the-first turn to
+avoid spamming the player
+- Removed "Average X" descriptions of species, leaving these only for
+the all-average humans
+- Made collapsing nebulae create various coloured stars
+- Added a "Rough Military Strength Estimate" empire statistic
+- Added Spinal Antimatter Cannon
+- Improved cross-platform consistency of random results in universe
+generation and in-game effects
+- Added Starlane Nexus and Starlane Bore buildings that can be used to
+create starlanes
+- Reworked Spatial Distortion Generator to push ships back along their
+approach starlane
+- Added Black Hole Collapser and Subspace Rift buildings
+- Adjusted some star names
+- Added sitreps to Planetary Starlane Drive effects
+- Various new hull art and slot positions, tech icons, building icons
+- Added the Accretion Disc special
+- Fixed / updated bioweapon sitreps
+- Added a random turn event that creates Space Krill in an empty system
+- Added new galaxy shape "Disc"
+- Added the feature to resurrect extinct species found in ancient
+ruins with the Xenoresurrection Lab
+- Added the new special Temporal Anomaly
+- Reworked weapon part refinements, so that instead of multiple parts,
```

Comment:
Seconded, I also wanted to suggest that for inclusion into the "Key Changes" section.